src/payment/data/handleRequestError.js

```diff
@@ -13,7 +35,13 @@ function handleFieldErrors(errors) {
   throw validationError;
 }
 
-function handleApiErrors(errors) {
+/**
+ * Process API Errors and Generate an Error Object
+ * @param {ApiErrorMessage[]} errors
+ * @param {boolean} shouldThrow
+ * @throws {Error} (Conditionally, but usually)
+ */
+export function generateApiError(errors, shouldThrow = true) {
```

Comment:
This was modified so it doesnt ALWAYS throw, if you tell it not to itll return the generated Error Object... Too many conditional catches tend to slow down execution as a runtime has to do a full stack trace.

Reply:
Update @throws message.